### ISSUE TITLE
Consider moving to official Jenkins plugin?

### ISSUE THREAD

**Issue by nwillems** [wenjunxiao/mixing-security-realm-plugin#2]
I was wondering if you, @wenjunxiao , were considering to commit this as an official jenkins plugin? From what I've read on tickets in the Jenkins jira space, it seems this feature is a long shot to be implemented natively.

I'll happily help in the process, but it seems ratehr easy to get going :-) This page provides nice info https://jenkins.io/doc/developer/guides/

**Comment by hxh-robb** [comment]
I agree with @nwillems , this is a good feature to publish to official plugin repository

**Comment by wenjunxiao** [comment]
Ok, I will publish to the official plugin repository.
